yt_dlp/extractor/jiocinema.py

```diff
@@ -0,0 +1,324 @@
+                ]
+            },
+            "continueWatchingRequired": True,
+            "dolby": False,
+            "downloadRequest": False,
+            "hevc": False,
+            "kidsSafe": False,
+            "manufacturer": "Mac OS",
+            "model": "Mac OS",
+            "multiAudioRequired": True,
+            "osVersion": "10.15.7",
+            "parentalPinValid": True
+        }
+
+        stream_response = self._download_json(url_or_request=stream_endpoint, video_id=None, note='Extracting Stream URL', data=bytes(json.dumps(payload), encoding='utf8'), headers=headers)
+
+        for url_data in stream_response['data']['playbackUrls']:
+            if url_data['encryption'] == 'widevine':
+                return url_data['url']
+
+    def _real_initialize(self):
+        super()._real_initialize()
+        if self.deviceId is None:
+            self.deviceId = str(uuid.uuid4())
+
+        self.auth_token = self._get_auth_token()
+
+    def _real_extract(self, url):
+        media_name, season_number, episode_name, media_id = re.match(self._VALID_URL, url).groups()
+
+        if not self.auth_token:
+            return
+
+        media_metadata = self._get_media_metadata(media_id)
+        mpd_url = self._get_stream_url(self.auth_token, media_id)
+
+        formats = []
+        formats.extend(self._extract_mpd_formats(mpd_url, media_id))
+
+        self._sort_formats(formats)
+        response_dict = {
+            'id': media_id,
+            'formats': formats,
+        }
+        response_dict.update(media_metadata)
+        return response_dict
+
+
+class JioCinemaTVIE(JioCinemaBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?jiocinema\.com/tv-shows/(?P<media_name>[a-zA-Z0-9\-]*)/(?P<season_number>[0-9]*)/(?P<episode_name>[a-zA-Z0-9\-]*)/(?P<media_id>[a-zA-Z0-9\-]*)'
+    _TEST = {
+        'url': 'https://www.jiocinema.com/tv-shows/cedric/2/mister-fixit/3769452',
+        'info_dict': {
+            'id': '3769452',
+            'ext': 'mp4',
+            'title': 'Mister Fixit',
+            'shortTitle': 'Mister Fixit',
+            'shortSynopsis': 'Cedric offers to fix Chen\'s broken walkman, however he realises that it is much harder than he thought it\'d be. He thinks of taking help from his father and Poppy, but th',
+            'fullSynopsis': 'Cedric offers to fix Chen\'s broken walkman, however he realises that it is much harder than he thought it\'d be. He thinks of taking help from his father and Poppy, but they get into an argument over who is a better handyman. Will Cedric be able to fix Chen\'s walkman as promised?',
+            'is4KSupported': False,
+            'is1080PSupported': False,
+            'isDolbySupported': False,
+            'introStart': 0,
+            'introEnd': 5,
+            'creditStart': 765,
+            'creditEnd': 770,
+            'recapStart': 0,
+            'recapEnd': 0,
+            'hasSubtitles': False,
+            'multiTrackAudioEnabled': False,
```

Comment:
infodict should only have standard fields as per https://github.com/yt-dlp/yt-dlp/blob/a174c453ee1e853c584ceadeac17eef2bd433dc5/yt_dlp/extractor/common.py#L119

Reply:
Fixed